### ISSUE TITLE
Include using unpkg.com

### ISSUE THREAD

**Issue by hanxue** [isagalaev/highlight.js#1612]
I am trying to use highlight.js via the [unpkg.com CDN](https://unpkg.com/highlight.js@9.12.0/). My HTML file has the following in the `<head>` section

```html
    <link href='//unpkg.com/highlight.js@latest/styles/solarized-dark.css' rel='stylesheet'/>
    <script src='//unpkg.com/highlight.js@latest' type='text/javascript'/>
    <script src='https://unpkg.com/highlight.js@latest/lib/languages/gradle.js' type='text/javascript'/>
    <script src='https://unpkg.com/highlight.js@latest/lib/languages/python.js' type='text/javascript'/>
    <script src='https://unpkg.com/highlight.js@latest/lib/languages/scala.js' type='text/javascript'/>
    <script src='https://unpkg.com/highlight.js@latest/lib/languages/r.js' type='text/javascript'/>
    <script type='text/javascript'>
      hljs.initHighlightingOnLoad();
```

All the files above, including jQuery, are successfully fetched from the CDN. However, highlight.js cannot run with the following error:

```
index.js:1 Uncaught ReferenceError: require is not defined
gradle.js:1 Uncaught ReferenceError: module is not defined
python.js:1 Uncaught ReferenceError: module is not defined
scala.js:1 Uncaught ReferenceError: module is not defined
r.js:1 Uncaught ReferenceError: module is not defined
create-and-publish-new-python-package.html:256 Uncaught TypeError: Cannot read property 'initHighlightingOnLoad' of undefined
    at create-and-publish-new-python-package.html:256
```

Am I missing something?

**Comment by Klooven** [comment]
I don't think you can use these files directly from the `lib` folder. (I've tried myself)

If you want to use languages that aren't included in the default release of highlightjs, like `Scala`, I suggest you to go and check the custom download page here: https://highlightjs.org/download/

Sadly I don't think it is possible to get custom highlightjs packages from any CDN.

**Comment by flxwu** [comment]
You actually can @hanxue using unpkg.

https://unpkg.com/highlight.js@9.12.0/lib/languages/gradle.js

for example works.

Go to 
https://unpkg.com/highlight.js@9.12.0/lib/languages/
to see all.

**Comment by yyyc514** [comment]
> Sadly I don't think it is possible to get custom highlightjs packages from any CDN.

No reason it wouldn't be possible but you'd have to package them correct for the browsers you were targeting and whatever JS module system you were using.

**Comment by yyyc514** [comment]
Ping.  Can someone still advise is there is an issue here or not?  **You can use the files on a CDN, but they need to be built for CDN usage** (ie, the same as the files we publish at the official URLs mentioned in the documentation).  You cannot just use the raw files from the GitHub repo or NPM. Well, maybe in some green-field browsers there is a way - but you'll have better luck using the existing CDN stuff I think.

You can build CDN packages from a git checkout with:

```
node tools/build.js -t cdn
```

The files you need at then in the `build` folder.  You could also just download all the files from the CDN, but that might be harder.  Or you need to use a build pipeline such as webpack, etc... that knows how to take the raw files and turn them into deployable assets.

**Comment by yyyc514** [comment]
Closing this issue from inactivity.  If there is still a question or issue we're happy to help and can reopen them.  (trying to close out all the stale issues that have long since been solved)
